examples/utils/example-utils/api-report/example-utils.api.md

```diff
@@ -346,7 +346,7 @@ export class SameContainerMigrationTool extends DataObject implements ISameConta
     // (undocumented)
     protected initializingFirstTime(): Promise<void>;
     // (undocumented)
-    get migrationState(): "collaborating" | "migrated" | "proposingMigration" | "stoppingCollaboration" | "proposingV2Code" | "waitingForV2ProposalCompletion" | "readyForMigration";
+    get migrationState(): "collaborating" | "proposingMigration" | "stoppingCollaboration" | "proposingV2Code" | "waitingForV2ProposalCompletion" | "readyForMigration" | "migrated";
```

Comment:
I think you should revert this. It really annoys me, but things just aren't stable. I get this all the time. :(
I think @tylerbutler has made efforts but we need something from api-extractor to get it really stable.